packages/http-server/src/AjvValidator.ts

```diff
@@ -20,19 +20,21 @@ const defaultAjvOptions: Options = {
 const sources = ['path', 'header', 'query', 'body'] as const;
 type Source = typeof sources[number];
 type AjvInstancesMap = Record<Source, Ajv>;
+type AjvOptionsMap = Record<Source, Options>;
 
 type AjvPluginOptions = unknown;
 type AjvPlugin = Plugin<AjvPluginOptions>;
+type AjvPlugins = Array<[AjvPlugin, AjvPluginOptions?]>;
+type AjvPluginsMap = Record<Source, AjvPlugins>;
 
 export interface AjvValidatorOptions {
-	ajvOptions?: Options;
-	plugins?: Array<[AjvPlugin, AjvPluginOptions?]>;
-	instances?: Ajv | Partial<AjvInstancesMap>;
+	ajvOptions?: Options | Partial<AjvOptionsMap>;
```

Comment:
I would rename this to `options`, for consistency.
Or both to `ajvOptions` and `ajvPlugins`, either is fine.